R/ggplotly.R

```diff
@@ -305,12 +305,28 @@ gg2list <- function(p){
       ax.list$tickangle <- -tick.text$angle
     }
     ax.list$tickfont <- theme2font(tick.text)
+
+    ## determine axis type first, since this information is used later
+    ## (trace.order.list is only used for type=category).
+    title.text <- e(s("axis.title.%s"))
+    ax.list$titlefont <- theme2font(title.text)
+    ax.list$type <- if(misc$is.continuous[[xy]]){
+      "linear"
+    }else if(misc$is.discrete[[xy]]){
+      "category"
+    }else if(misc$is.date[[xy]] || misc$is.datetime[[xy]]){
+      "date"
+    }else{
```

Comment:
I'm assuming `ggplot2` doesn't have a `log` type? (that it is just `continuous`)